colorblindfriendly.py

```diff
@@ -37,15 +37,14 @@
 
     # Replace built-in colors with cbf ones
     cbf.set_colors(replace=True)
-    color my_other_object, yellow   # actually cb_yellow
+    color myOtherObject, yellow   # actually cb_yellow
 
     # Add a cb_colors menu item to the OpenGL GUI ([C] menu in the right panel)
-    # Note: `__main__` argument is required for this to work!
-    cbf.add_menu(__main__)
+    cbf.add_menu()
 
 REQUIREMENTS
 
-    None.
+    The `add_menu()` function is only available for PyMOL 2.0 and later.
```

Comment:
works for me in 1.7.4

also works in 1.5.0 after the following modification to the `add_menu()` function:

```
    # Check for cb_colors
    print('Checking for colorblindfriendly colors...')
    try:
        if cmd.get_color_index('cb_red') == -1:
            raise pymol.CmdException
    except pymol.CmdException:
        print('Adding colorblindfriendly colors...')
        set_colors()

    try:
        from pymol.menu import all_colors_list
    except ImportError:
        print('PyMOL version too old for cb_colors menu')
        return
```

Reply:
Thanks for this.  So... you raise the exception instead of just using the -1 return value because `cmd.get_color_index()` raised an exception prior to 1.7.4 if the color didn't exist?  And 1.6 introduced `pymol.menu.all_colors_list`.  Nice!

Reply:
I was initially looking at overriding `pymol.menu.all_colors()` but found that it wasn't present in 2.0.  I assumed that `all_colors_list` was new, but I guess not!